packages/schematics/angular/utility/ng-ast-utils.ts

```diff
@@ -12,7 +12,7 @@ import { dirname } from 'path';
 import * as ts from '../third_party/github.com/Microsoft/TypeScript/lib/typescript';
 import { findNode, getSourceNodes } from '../utility/ast-utils';
 
-export function findBootstrapModuleCall(host: Tree, mainPath: string): ts.CallExpression | null {
```

Comment:
While this is technically not public API, it is still accessible.  We may want to leave it exported for now and drop it in v16 just to avoid any potential breakage.